packages/devp2p/src/protocol/protocol.ts

```diff
@@ -4,9 +4,10 @@ import { EventEmitter } from 'events'
 import { devp2pDebug } from '../util'
 import { Peer, DISCONNECT_REASONS } from '../rlpx/peer'
 
-export enum EthProtocol {
+export enum EthProtocol { // What does this represent?
```

Comment:
This represents the protocol running on top of RLPx. Could probably just be renamed to `Protocol`, but that may conflict with the base class? `RlpxProtocol` could also work, but not sure if we want to rename/breaking change.

Reply:
We could rename this enum to `ProtocolType`?